tests/enterprise/bulkTransfer.test.ts

```
@@ -0,0 +1,409 @@
+      organizationId: "org-1",
+      senderUserId: "user-1",
+      fileContent: Buffer.from(csv),
+    });
+
+    expect(result.successCount).toBe(1);
+    expect(result.skippedCount).toBe(1);
+    expect(createdTransactions.size).toBe(1);
+  });
+
+  it("does not log recipient identifiers or transfer amounts", async () => {
+    const csv = makeCsv([{ to: "recipient-secret", amount_acbu: "999.123", reference: "ref-a", idempotency_key: "key-a" }]);
```

ESLint

[error] Replace `{·to:·"recipient-secret",·amount_acbu:·"999.123",·reference:·"ref-a",·idempotency_key:·"key-a"·}` with `⏎······{⏎········to:·"recipient-secret",⏎········amount_acbu:·"999.123",⏎········reference:·"ref-a",⏎········idempotency_key:·"key-a",⏎······},⏎····`

(prettier/prettier)